.github/workflows/broadcast-frontend-hash.yml

```
@@ -0,0 +1,70 @@
+name: Broadcast Frontend Hash
+on:
+  workflow_dispatch:
+    inputs:
+      dfx_version:
+        description: 'Release version of dfx'
+        default: "latest"
+  # release:
+  #   # https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=released#release
+  #   # "A release was published, or a pre-release was changed to a release."
+  #   types: [released]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  PLAYGROUD_REPO: ${{ github.repository_owner }}/motoko-playground
+  FILE: service/wasm-utils/lib.rs
+
+jobs:
+  update-frontend-hash:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sdk repo
+        uses: actions/checkout@v3
+        with:
+          path: sdk
+
+      - name: Checkout playground repo
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.NIV_UPDATER_TOKEN }}
+          repository: ${{ env.PLAYGROUD_REPO }}
+          path: motoko-playground
+
+      - name: Get new hash
+        working-directory: ./sdk
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          export NEW_HASH=$(.github/workflows/broadcast-frontend-hash_extract-hash-from-release-notes.sh ${{ inputs.dfx_version }})
```

shellcheck warns about getting the results of a subshell and exporting it on the same line, because it ignores errors.

But also, in the context of a github workflow where we are appending this to `$GITHUB_ENV` anyway, we don't need to export at all.

```suggestion
          NEW_HASH=$(.github/workflows/broadcast-frontend-hash_extract-hash-from-release-notes.sh ${{ inputs.dfx_version }})
```